src/constants/editor.ts

```diff
@@ -116,6 +116,11 @@ const EDITOR_BASE_CONTEXT_SUGGESTIONS: CodeEditorSuggestionItem[] = [
     kind: CodeEditorSuggestionItemKind.Method,
     detail: 'Show error notification.',
   },
+  {
+    label: 'context.grafana.refresh',
+    kind: CodeEditorSuggestionItemKind.Method,
+    detail: 'Interpolate all variables.',
```

Comment:
Please update details to https://github.com/VolkovLabs/volkovlabs-form-panel/blob/d26cdca41b144105f09bd0decb29d62cd09cc163/src/constants/code-editor.ts#L218

Reply:
updated